src/currency.js

```diff
@@ -131,14 +141,25 @@ Globalize.prototype.formatCurrency = function( value, currency, options ) {
  *
  * @value [String]
  *
- * @currency [String] 3-letter currency code as defined by ISO 4217.
- *
  * @options [Object]: See currencyFormatter.
  *
  * Return the parsed currency or NaN when value is invalid.
  */
 Globalize.parseCurrency =
-Globalize.prototype.parseCurrency = function( /* value, currency, options */ ) {
+Globalize.prototype.parseCurrency = function( value, options ) {
+	validateParameterPresence( value, "value" );
+	value = value.replace( /[^\d\.\,\-]/g, "" );
+	if ( value[0] === "-" ) {
+		value = value.replace( "-", "" );
+		value = "-".concat( value );
+	} else {
+		value = value.replace( "-", "" );
+	}
```

Comment:
The above digits and minus sign should be taken care by numberParser below. PS: Note that `/\d/` doesn't work on other numbering systems, e.g., arabic. It's not necessary to check for that anyway (numberParser will take care)

Reply:
I actually *didn't* know `\d` won't work with non-ASCII numbers. How do I obtain the regex for those digits? Regarding this section in particular, I guess I wanted to make sure I didn't leave any `-`, but I'm realizing now that if there's a dash in the middle, it should return `NaN`. I'll take that out.

Reply:
I can use `/\p{Decimal_Number}+/u` instead, but that requires ES2018

Reply:
If you need something more dynamic and that's supported cross-browser you could use `new RegExp()`

Reply:
can you show me where I would add `new RegExp()`?